tests/test_core/test_alignment.py

```diff
@@ -2157,6 +2183,52 @@ def test_to_html(self):
         # order now changes
         self.assertTrue(got.find(ref_row) < got.find(other_row))
 
+    def test_to_html_deprecation_warning(self):
+        """ should raise warning using wrap and not interleave_len"""
+        seqs = {"seq1": "ACG", "seq2": "-CT"}
+
+        aln = self.Class(data=seqs, moltype=DNA)
+        got = aln.to_html(ref_name="longest")  # name_order=['seq1', 'seq2'])
+        # ensure balanced tags are in the txt
+        for tag in ["<style>", "</style>", "<div", "</div>", "<table>", "</table>"]:
+            self.assertTrue(tag in got)
+
+        ref_row = (
+            '<tr><td class="label">seq1</td>'
+            '<td><span class="A_dna">A</span>'
+            '<span class="C_dna">C</span>'
+            '<span class="G_dna">G</span></td></tr>'
+        )
+        other_row = (
+            '<tr><td class="label">seq2</td>'
+            '<td><span class="ambig_dna">-</span>'
+            '<span class="C_dna">.</span>'
+            '<span class="T_dna">T</span></td></tr>'
+        )
+
+        self.assertTrue(ref_row in got)
+        self.assertTrue(other_row in got)
+        self.assertTrue(got.find(ref_row) < got.find(other_row))
+
+        # using different ref sequence
+        ref_row = (
+            '<tr><td class="label">seq2</td>'
+            '<td><span class="terminal_ambig_dna">-</span>'
+            '<span class="C_dna">C</span>'
+            '<span class="T_dna">T</span></td></tr>'
+        )
+        other_row = (
+            '<tr><td class="label">seq1</td>'
+            '<td><span class="A_dna">A</span>'
+            '<span class="C_dna">.</span>'
+            '<span class="G_dna">G</span></td></tr>'
+        )
+        # specify interleave_len in 2 cases, wrap specified and not specified
+        got_no_wrap = aln.to_html(ref_name="seq2", interleave_len=40)
+        got_wrap = aln.to_html(ref_name="seq2", interleave_len=40, wrap=50)
+        # raise warnings
+        self.assertRaises(DeprecationWarning)
```

Comment:
if the test is just checking the deprecation warning is being raised, the use the `with` context statement as above to simplify this test.